### DISCUSSION TITLE
Reusing Intellichem Dosing Pumps on 15 gallon tanks

### DISCUSSION THREAD

**Discussion by davthomaspilot** [tagyoureit/nodejs-poolController#765] [REM]
I spent quite a bit of time iterating on a design for mounting an Intellichem dosing pump on a 15 gallon tank.  If anyone wants a .stl for 3d printing it, let me know and I'll email it.

Background....

I spend a month in Montana each summer.  I don't want the house sitter to have to refill the tank(s) while I'm away, so I wanted a bigger chlorine tank than what I got with my Pentair Intellichem setup (4 gallons).

I saw the Stenner 15 gallon tank that's similar to this:

https://www.freshwatersystems.com/products/stenner-15-gallon-tank-for-45-and-85-series-pumps?variant=37074785730744&c1=GAW_SE_NW&source=PLA_USA_PM&cr2=pmax__-__nw__-__shopping&kw=37074785730744&cr5=pla&cr7=c&utm_term=37074785730744&utm_campaign=pmax__-__nw__-__shopping&utm_source=PLA_USA_PM&utm_medium=cpc&hsa_acc=2014103020&hsa_cam=11853986621&hsa_grp=&hsa_ad=&hsa_src=x&hsa_tgt=&hsa_kw=&hsa_mt=&hsa_net=adwords&hsa_ver=3&gclid=CjwKCAjwxr2iBhBJEiwAdXECw2uzhrJrqcBB9VwHZq06UFRsXWZghTVBnUivcss_iOSoDOD85eBmLxoCuBUQAvD_BwE

The Intellichem dosing pumps look very similar to the Stenner "Econ" series of pumps.  They use the same pump tubes and head assemblies.  But, I talked to Stenner technical staff and was told that I couldn't to reuse the Intellichem pumps with the 15 gallon tank.  While Pentair does purchase the pumps from Stenner, they are specially designed for integration into the Intellichem tanks and there "is no way" to use them with the 15 gallon tanks.

But, I figured I could 3d print an enclosure for mounting the Intellichem pumps...

Here's what the Intellichem looks like with the pump removed (8 screws):

![20230517_171943](https://github.com/tagyoureit/nodejs-poolController/assets/6830886/d677847a-7252-4f40-8e7a-6534c57eb83d)

![20230517_171901 (1)](https://github.com/tagyoureit/nodejs-poolController/assets/6830886/43d3efe0-bfbd-40a5-a5d3-62e01a9102ad)

Part of the enclosure is removed by four screws, but the pump motor extrudes so that a "female" enclosure/mount with enough depth is needed:
![20230517_172021](https://github.com/tagyoureit/nodejs-poolController/assets/6830886/8050f346-87e1-4d9c-96fd-8f5f4eb7e216)

I designed an enclosure that would mount on the tank and have the "lip" and required depth:

![image](https://github.com/tagyoureit/nodejs-poolController/assets/6830886/7443ae99-a82a-41d5-ad9e-fc9321fbe6fc)

Here's what it looks like on the tank, along with the enclosure part that is removed from the tank:

![20230522_142442](https://github.com/tagyoureit/nodejs-poolController/assets/6830886/5355a089-626a-46b5-b6c2-8f4b8f2c04cb)

Then, the pump is attached using four screws i.

**Comment by rstrouse** [comment]
And I begin another session of looking at 3d printers and days of talking myself out of it.  Curse your foul hide.

**Reply by davthomaspilot** [reply]
Best (and certainly most fun!) investment I've done. 

Last week, I saved $70 by printing a drain plug for an irrigation pump (yeah, $70 for stupid 1/4" plug--it wasn't NPT threads, but found that 18 mm DIN fit perfectly.

Can't tell you how many times it came in handy.

But, be prepared to waste? lots of time playing for months to come.

Do it!
